### DISCUSSION TITLE
Be able to use Elementary OS in a professional context

### DISCUSSION THREAD

**Discussion by SpiralisT** [elementary/triage#378] [Ideas]
Dears, 

I think it would be interesting to consider adding the missing elements to integrate Elementary OS as End-Points in a global company infrastructure, and have it available out-of-the-box.

What I have in mind is for instance:
- VPN Connections (to Fortigates, Cisco, or whatever) ;
- ActiveDirectory integration;
- End-Point management agents (for automatic deployments, centralized inventory, ...)

I use Elementary OS in a private context for a few months now, and I am impressed with its stability. But when I tried to configure it to integrate it as an end-point of my Company infrastructure, some components were missing (not able to install a FortiGate VPN Client, not able to easily integrate into an AD domain).

Maybe it is time to ramp up and consider this?
Cheers

**Comment by jlnr** [comment]
I mostly use elementary for work (personal life is in Apple land), and I have also had trouble with e.g. OpenVPN. I know that these are all really boring things to work on, but on the other hand, if there was a way to support elementary as a business, it would be more tax efficient.
